### PR TITLE
Rtd setup

### PR DESCRIPTION
**Pull request by FlorianWilhelm**
Just applying some of the goodies from PyScaffold v4 to this extension.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -76,13 +78,15 @@
     "sphinx.ext.ifconfig",
     "sphinx.ext.mathjax",
     "sphinx.ext.napoleon",
+    "sphinx.ext.extlinks",
+    "recommonmark",
 ]
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
```
Comment (abravalheri) [inline docs/conf.py:87]: Just curiosity... Is AutoStructify setup required here? To be sincere I don't know exactly what it does...

**Review by abravalheri — APPROVED** [review]

**Comment by abravalheri** [comment]
Thank you very much for the updates Florian, looks very good.

File: docs/conf.py (hunk 1 above)
Comment (FlorianWilhelm) [reply docs/conf.py:87]: Good point. I added it as it allows as it activates some more useful features like inline math in md docs.

Comment (abravalheri) [reply docs/conf.py:87]: That is great @FlorianWilhelm, feel free to merge the commit when you are satisfied :)

**Pull request merged**